storm-core/src/clj/backtype/storm/daemon/nimbus.clj

```diff
@@ -205,6 +249,10 @@
                       (log-message "Killing topology: " storm-id)
                       (.remove-storm! (:storm-cluster-state nimbus)
                                       storm-id)
+                      (if (instance? LocalFsBlobStore (:blob-store nimbus))
```

Comment:
We do not want to create a state for hdfs blobstore as the interaction between the daemons (nimbus and supervisor) with hdfs is direct and CRUD operations are handled by HDFS